### PR TITLE
Add some attributes.

### PR DESCRIPTION
**Pull request by ton-katsu**
- Organization
  - default_repository_permission
  - has_organization_projects
  - has_repository_projects
  - hooks_url
  - issues_url
  - members_can_create_repositories
- Repository
  - deployments_url
  - has_pages
  - releases_url

### CODE REVIEW

**Comment by s-t-e-v-e-n-k** [comment]
This looks like a great start! If you add tests for the rest of the attributes, we can get this landed.

**Comment by ton-katsu** [comment]
> This looks like a great start! If you add tests for the rest of the attributes, we can get this landed.
Could you give me an example where should I update?

**Comment by s-t-e-v-e-n-k** [comment]
tests/Organization.py should exist for checking any of the attributes you've added. But also, you add has_pages to Repository, and it isn't tested.

**Comment by ton-katsu** [comment]
What do you think about added commits?

File: tests/Organization.py [hunk 1]
```diff
@@ -292,6 +307,8 @@ def testCreateRepoWithMinimalArguments(self):
         self.assertEqual(
             repo.url, "https://api.github.com/repos/BeaverSoftware/TestPyGithub"
         )
+        self.assertEqual(repo.has_wiki, True)
```
Comment (s-t-e-v-e-n-k) [inline tests/Organization.py:310]: self.assertTrue(repo.has_wiki)

File: tests/Organization.py [hunk 2]
```diff
@@ -313,6 +330,8 @@ def testCreateRepoWithAllArguments(self):
         self.assertEqual(
             repo.url, "https://api.github.com/repos/BeaverSoftware/TestPyGithub2"
         )
+        self.assertEqual(repo.has_wiki, False)
+        self.assertEqual(repo.has_pages, False)
```
Comment (s-t-e-v-e-n-k) [inline tests/Organization.py:334]: self.assertFalse(repo.has_wiki)

File: tests/Organization.py [hunk 3]
```diff
@@ -321,13 +340,17 @@ def testCreateRepositoryWithAutoInit(self):
...
+        self.assertTrue(repo.has_wiki)
 
     def testCreateFork(self):
         pygithub = self.g.get_user("jacquev6").get_repo("PyGithub")
         repo = self.org.create_fork(pygithub)
         self.assertEqual(
             repo.url, "https://api.github.com/repos/BeaverSoftware/PyGithub"
         )
+        self.assertFalse(repo.has_wiki, False)
+        self.assertFalse(repo.has_pages, False)
```
Comment (s-t-e-v-e-n-k) [inline tests/Organization.py:353]: How is this not a syntax error, wow. Anyway, remove the unneeded second argument.

File: tests/Organization.py [hunk 4]
```diff
@@ -367,11 +390,9 @@ def testInviteUserAsNonOwner(self):
...
     def testCreateMigration(self):
         self.org = self.g.get_organization("sample-test-organisation")
-        self.assertTrue(
-            isinstance(
-                self.org.create_migration(["sample-repo"]), github.Migration.Migration
-            )
-        )
+        migration = self.org.create_migration(["sample-repo"])
+        self.assertTrue(isinstance(migration, github.Migration.Migration))
+        self.assertListKeyEqual(migration.repositories, lambda r: r.has_pages, [False])
```
Comment (s-t-e-v-e-n-k) [inline tests/Organization.py:395]: Nah, this is fine to leave as is, you don't need to check it in every place, and this test is already a little awkward.

**Comment by ton-katsu** [comment]
Sorry, so easy mistake.
Could you review this code again?

**Comment by ton-katsu** [comment]
@s-t-e-v-e-n-k 
Thank you for approval.
When do you merge this PR?

**Pull request merged**